packages/react-components/lib/task-summary/task-summary-accordion.tsx

```diff
@@ -0,0 +1,179 @@
+import { IconButton, makeStyles, Typography } from '@material-ui/core';
+import { TreeItem, TreeView } from '@material-ui/lab';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import PlayCircleFilledWhiteIcon from '@material-ui/icons/PlayCircleFilledWhite';
+import PauseCircleFilledIcon from '@material-ui/icons/PauseCircleFilled';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import { SimpleInfo, SimpleInfoData } from '../simple-info';
+import { formatStatus, getActorFromStatus, getStateLabel } from './task-summary-utils';
+
+const debug = Debug('Tasks:TaskSummaryAccordion');
+
+interface TaskSummaryAccordionInfoProps {
+  task: RomiCore.TaskSummary;
+}
+
+export const TaskSummaryAccordionInfo = (props: TaskSummaryAccordionInfoProps): JSX.Element => {
+  const { task } = props;
+  const statusDetails = formatStatus(task.status);
+  const stateLabel = getStateLabel(task.state);
+  const data = [
+    { name: 'TaskId', value: task.task_id, wrap: false },
+    { name: 'State', value: stateLabel },
+    { name: 'Status', value: statusDetails },
+    { name: 'Submission Time', value: task.submission_time.sec },
+    { name: 'Start time', value: task.start_time.sec },
+    { name: 'End Time', value: task.end_time.sec },
+  ] as SimpleInfoData[];
+
+  return <SimpleInfo infoData={data} />;
+};
+
+export interface TaskSummaryAccordionProps {
+  tasks: RomiCore.TaskSummary[];
+}
+
+export const TaskSummaryAccordion = React.memo((props: TaskSummaryAccordionProps) => {
+  debug('task summary status panel render');
+  const { tasks } = props;
+  const classes = useStyles();
+  const [expanded, setExpanded] = React.useState<string[]>([]);
+  const [selected, setSelected] = React.useState<string>('');
+
+  const handleToggle = (event: React.ChangeEvent<Record<string, unknown>>, nodeIds: string[]) => {
+    setExpanded(nodeIds);
+  };
+
+  const handleSelect = (event: React.ChangeEvent<Record<string, unknown>>, nodeIds: string) => {
+    setSelected(nodeIds);
+  };
+
+  const determineStyle = (state: number): string => {
+    switch (state) {
+      case RomiCore.TaskSummary.STATE_QUEUED:
+        return classes.queued;
+      case RomiCore.TaskSummary.STATE_ACTIVE:
+        return classes.active;
+      case RomiCore.TaskSummary.STATE_COMPLETED:
+        return classes.completed;
+      case RomiCore.TaskSummary.STATE_FAILED:
+        return classes.failed;
+      default:
+        return 'UNKNOWN';
+    }
+  };
+
+  const renderActor = (taskStatus: string): React.ReactElement | null => {
+    const actor = getActorFromStatus(taskStatus);
+    if (!actor) return null;
+    return (
+      <Typography variant="body1" id="task-actor" className={classes.taskActor}>
+        {actor}
+      </Typography>
+    );
+  };
+
+  const renderTaskTreeItem = (task: RomiCore.TaskSummary) => {
+    return (
+      <TreeItem
+        data-component="TreeItem"
```

Comment:
Avoid using `data-component`, see if there is any suitable aria roles (listitem or row maybe?) for this. Need to change the e2e tests as well after we changed this.

Reply:
Ok, I'm using a role now, `role='treeitem'`